Jenkinsfile

```diff
@@ -191,6 +194,70 @@ pipeline {
                         }
                     }
                 }
+
+                stage ('Build Docker Native Dependency') {
+                    when {
+                        environment name: 'BUILD_PRESTISSIMO_DEPENDENCY', value: 'true'
+                    }
+                    steps {
+                        sh '''
+                            cd presto-native-execution/
+                            make submodules
+                            docker buildx build --load --platform "linux/amd64" \
+                                    -t "${NATIVE_DOCKER_IMAGE_DEPENDENCY}" \
+                                    --build-arg BUILD_TYPE=Release \
```

Comment:
There is no build arg to the dependency dockerfiles
https://github.com/prestodb/presto/blob/master/presto-native-execution/scripts/dockerfiles/centos-8-stream-dependency.dockerfile

Reply:
removed